pykickstart/commands/raid.py

```diff
@@ -414,16 +499,46 @@ def __init__(self, writePriority=131, *args, **kwargs):
 
         self.levelMap.update({"RAID10": "RAID10", "10": "RAID10"})
 
+    def _getParser(self):
+        op = FC5_Raid._getParser(self)
+        for action in op._actions:
+            if "--level" in action.option_strings:
+                action.help += dedent("""
+
+                .. versionchanged:: %s
+
+                The "RAID10" level was added.""" % \
+                versionToLongString(F7))
+                break
+        return op
+
 class F9_Raid(F7_Raid):
     removedKeywords = F7_Raid.removedKeywords
     removedAttrs = F7_Raid.removedAttrs
 
     def _getParser(self):
         op = F7_Raid._getParser(self)
-        op.add_argument("--bytes-per-inode", deprecated=True)
-        op.add_argument("--fsprofile")
-        op.add_argument("--encrypted", action="store_true", default=False)
-        op.add_argument("--passphrase")
+        op.add_argument("--fsprofile", version=F9, help="""
+                        Specifies a usage type to be passed to the program that
+                        makes a filesystem on this partition. A usage type
+                        defines a variety of tuning parameters to be used when
+                        making a filesystem. For this option to work, the
+                        filesystem must support the concept of usage types and
+                        there must be a configuration file that lists valid
+                        types. For ext2/3/4, this configuration file is
+                        ``/etc/mke2fs.conf``.""")
+        op.add_argument("--encrypted", action="store_true", version=F9,
+                        default=False, help="""
+                        Specify that this RAID device should be encrypted.""")
+        op.add_argument("--passphrase", version=F9, help="""
+                        Specify the passphrase to use when encrypting this RAID
+                        device. Without the above --encrypted option, this option
+                        does nothing. If no passphrase is specified, the default
+                        system-wide one is used, or the installer will stop and
+                        prompt if there is no default.""")
+#todo: it looks like this needs to be removed earlier, see F9_RaidData
+# instead of F14. Test fails otherwise. Why doesn't it fail in Jenkins ?
```

Comment:
OK, I've removed this (and related comments). Can you kick off a Jenkins build (b/c I can't) and see if this passes. I will later investigate why it fails for me locally.